api/external/envoy/api/v2/core/health_check.proto

```diff
@@ -0,0 +1,300 @@
+syntax = "proto3";
```

Comment:
Gloo's proto was replaced with this one https://github.com/solo-io/gloo/pull/4777/files#r639104352